### PR TITLE
[Docs] Filter Cog Guide

### PR DESCRIPTION
**Pull request by bobloy**
### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes

User guide for the Filter cog. Docs were automatically generated using the same method as #4488 
A step towards the #1734 goal

### CODE REVIEW

**Comment by bobloy** [comment]
This is now ready for review. If #4599 is accepted, will fix `[words...]` to `<words...>`

File: docs/cog_guides/filter.rst [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+        [p]load filter
+
+.. _filter-usage:
+
+-----
+Usage
+-----
+
+This cog is designed for "filtering" unwanted words and phrases from a guild.
```
Comment (Flame442) [inline docs/cog_guides/filter.rst]:
```suggestion
This cog is designed for "filtering" unwanted words and phrases from a server.
```
Everywhere else in this document, "server" is used, so either consistently use "guild" or change this instance.

File: docs/cog_guides/filter.rst [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]filter 
+
+**Description**
+
+Base command to add or remove words from server filter.
```
Comment (Flame442) [inline docs/cog_guides/filter.rst]:
```suggestion
Base command to add or remove words from the server filter.
```
I'm also not a fan of "*add* or *remove* words *from*", but there isn't a good way to make this sound better besides being overly verbose ("add words to or remove words from"). Unless you can think of a better wording I'm fine leaving that how it is.

File: docs/cog_guides/filter.rst [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]filter channel 
+
+**Description**
+
+Base command to add or remove words from channel filter.
```
Comment (Flame442) [inline docs/cog_guides/filter.rst]:
```suggestion
Base command to add or remove words from the channel filter.
```

File: redbot/cogs/filter/filter.py [hunk 4]
```diff
@@ -14,7 +14,12 @@
 
 @cog_i18n(_)
 class Filter(commands.Cog):
-    """Filter unwanted words and phrases from text channels."""
+    """This cog is designed for "filtering" unwanted words and phrases from a guild.
```
Comment (Flame442) [inline redbot/cogs/filter/filter.py]:
```suggestion
    """This cog is designed for "filtering" unwanted words and phrases from a server.
```

File: redbot/cogs/filter/filter.py [hunk 5]
```diff
@@ -114,15 +135,15 @@ async def filter_ban(self, ctx: commands.Context, count: int, timeframe: int):
     @commands.guild_only()
     @checks.mod_or_permissions(manage_messages=True)
     async def _filter(self, ctx: commands.Context):
-        """Add or remove words from server filter.
+        """Base command to add or remove words from server filter.
```
Comment (Flame442) [inline redbot/cogs/filter/filter.py]:
```suggestion
        """Base command to add or remove words from the server filter.
```

File: redbot/cogs/filter/filter.py [hunk 6]
```diff
@@ -139,15 +160,15 @@ async def _global_list(self, ctx: commands.Context):
 
     @_filter.group(name="channel")
     async def _filter_channel(self, ctx: commands.Context):
-        """Add or remove words from channel filter.
+        """Base command to add or remove words from channel filter.
```
Comment (Flame442) [inline redbot/cogs/filter/filter.py]:
```suggestion
        """Base command to add or remove words from the channel filter.
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
Some minor consistency and grammar issues, but otherwise solid. Thanks for contributing.

**Review by Flame442 — DISMISSED** [review]
Good to go once the merge conflict is resolved!

**Review by Flame442 — APPROVED** [review]
📦

**Pull request merged**